packages/competence/deploy/gcp/WALKTHROUGH.md

```
@@ -122,26 +122,56 @@ BUDGET_NAME="<your budget's display name>" ./bootstrap.sh   # recognise the exis
+        acceptable only because IAP decides who reaches the app at all — see phase 7, and do not skip it.
+- [ ] **The app's client.** Clients → **Create client**, type **Web application**, name it e.g.
+      `competence app sign-in`. **Leave the redirect URI empty** — phase 7 fills it in, once the
+      service URL exists.
 - [ ] Copy both values. The **Client ID** ends in `.apps.googleusercontent.com` and is not sensitive.
       The **Client secret** is — do not put it in a file, a chat, or a commit.
+- [ ] **IAP's client — only if the project has no organization.** Create a *second* **Web
+      application** client, named e.g. `competence IAP`. Then reopen it and add this authorised
+      redirect URI, which contains **that client's own ID**, so it can only be filled in after the
+      client exists:
+
+      ```text
```

markdownlint-cli2 (0.23.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)